.github/workflows/build.yml

```diff
@@ -11,26 +11,36 @@ on:
 
 jobs:
   build-sentry-native:
-    name: sentry-native (${{ matrix.os }})
+    name: sentry-native (${{ matrix.os }}${{ matrix.arch && format('-{0}', matrix.arch) || '' }}
```

Comment:
perhaps this change can be reverted. the name becomes `sentry-native (windows-11-arm-arm64`

- the arch suffix is not necessary in this context
- missing closing parenthesis

Reply:
Done

![image](https://github.com/user-attachments/assets/9db76434-af60-446c-a964-4a3ddaf80739)